zerver/data_import/slack.py

```diff
@@ -862,7 +863,7 @@ def convert_slack_workspace_messages(
 
         total_reactions += reactions
         total_attachments += attachment
-        total_uploads += uploads
+        total_uploads += [asdict(upload) for upload in uploads]
```

Comment:
I'm curious why we can't propagate this further, without the `asdict`.

Regardless, certainly feels like an improvement.